package.json

```diff
@@ -14,8 +14,7 @@
     "@typescript-eslint/eslint-plugin": "^2.7.0",
     "@typescript-eslint/parser": "^2.7.0",
     "eslint": "^6.6.0",
-    "lerna": "^3.18.4",
-    "typescript": "~3.6.4"
+    "lerna": "^3.18.4"
```

Comment:
wouldn't it make sense to remove all dependencies from the root?

Reply:
Probably would. I'll do this in a separate PR, though.